independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/README.md

```diff
@@ -0,0 +1,36 @@
+# Open Trivia Database
+
+The Open Trivia DB is a fun and interactive API that provides an easy way to acquire fun and random trivia questions at a moments notice.
+
+## Publisher: Kiveshan Naidoo, Rishay Sunder
+
+
+## Prerequisites
+
+None
+
+## Supported Operations
+
+### Get a certain number of trivia questions, based on a criteria
+
+Get a certain amount of questions from the database based on categories such as diffuculty, Question Type and Category. Leaving
```

Comment:
@Kiveshan please make sure that all files are free of grammatical errors. here you have "diffuculty" and I think you meant to say "difficulty".